src/components/CardPage.module.css

```diff
@@ -1,26 +1,47 @@
+.bottom {
```

Comment:
not worth changing at this point but .bottom seems like a very vague name considering that it is doing semingly very specific positioning